support/testwizard/main.py

```diff
@@ -0,0 +1,331 @@
+  assets = await openspace.asset.rootAssets()
+  folder = await openspace.absPath("${ASSETS}")
+  assets = [asset[len(folder)+1:asset.find(".")] for index,asset in assets.items()]
+
+  print("List of assets:")
+  for index,asset in enumerate(assets):
+    print(  f"  ({index+1}): {asset}")
+
+  asset = ""
+  while not asset.isnumeric():
+    asset = input("Select which asset to load: ")
+
+    if asset == "":
+      return None
+
+    idx = int(asset) - 1
+
+    if idx < 0 or idx >= len(assets):
+      print("Invalid index")
+      continue
+
+  asset = assets[idx]
+  asset = asset.replace("\\", "/")
+
+  return {
+    "type": "asset",
+    "value": f"{asset}.asset"
+  }
+
+
+async def createCommandProperty(openspace):
+  uri = input("URI of the property: ")
+  if uri == "":
+    return None
+
+  if uri.find("*") != -1 or uri.find("{") != -1:
+    # The URI refers to multiple property values, so we need to as for the specific value
+    value = input("Enter value of the property: ")
+
+    # Perform some type conversions if necessary
+    if value.lower() == "true":
+      value = True
+    elif value.lower() == "false":
+      value = False
+
+    if value.isnumeric():
+      value = int(value)
+
+  else:
+    has_property = await openspace.hasProperty(uri)
+    if not has_property:
+      print(f"Property '{uri}' not found")
+      return None
+
+    # This is a property identifier that we can use to request a value for
+    value = await openspace.propertyValue(uri)
+
+  return {
+    "type": "property",
+    "value": {
+      "property": uri,
+      "value": value
+    }
+  }
+
+
+async def createCommandWait(openspace):
+  wait = input("Number of seconds to wait: ")
+  if wait == "":
+    return None
+
+  return {
+    "type": "wait",
+    "value": wait
+  }
+
+
+async def createCommandScript(openspace):
+  script = input("Script that should be executed: ")
+
+  return {
+    "type": "script",
+    "value": script
+  }
+
+
+async def createCommandTime(openspace):
+  time = await openspace.time.UTC()
+
+  return {
+    "type": "time",
+    "value": time
+  }
+
+
+async def createCommandPause(openspace):
+  pause = input("New pause state: (y/n). Default 'y': ")
+  pause = pause or "y"
+
+  p = ""
+  if pause == "y":
+    p = "true"
+  else:
+    p = "false"
+
+  return {
+    "type": "pause",
+    "value": p
+  }
+
+
+async def createCommandDeltatime(openspace):
+  dt = await openspace.time.deltaTime()
+
+  return {
+    "type": "deltatime",
+    "value": dt
+  }
+
+
+async def createCommandAction(openspace):
+  actions = await openspace.action.actions()
+  actions = [action for index,action in actions.items()]
+
+  print("List of actions:")
+  for index,action in enumerate(actions):
+    print(  f"  ({index+1}): {action["Name"]} ({action["Identifier"]})")
+
+  action = ""
+  while not action.isnumeric():
+    action = input("Select which action to execute: ")
+
+    if action == "":
+      return None
+
+    idx = int(action) - 1
```

Comment:
Same here, maybe it could be clearer that the user is supposed to enter an index here.